modules/settings/settings.php

```diff
@@ -221,10 +221,10 @@ function print_default_footer( $current_module ) {
 		?>
 		<?php if ( $current_module->slug == 'settings' ): ?>
 		<div class="credits">
-		<p><a href="http://editflow.org/">Edit Flow</a> is produced by <a href="http://danielbachhuber.com/">Daniel Bachhuber</a>, <a href="http://digitalize.ca/">Mo Jangda</a>, and <a href="http://www.scottbressler.com/blog/">Scott Bressler</a>, with special help from <a href="http://andrewspittle.net">Andrew Spittle</a> and <a href="http://andrewwitherspoon.com/">Andrew Witherspoon</a>.
-		<br />You're using Edit Flow v<?php echo EDIT_FLOW_VERSION; ?>.
-		<br />Icons courtesy of the <a href="http://thenounproject.com/">Noun Project</a>.
-		<br /><a href="http://wordpress.org/tags/edit-flow?forum_id=10">Please give us your feedback, ideas, bug reports and comments</a> in the WordPress.org forums.
+		<p><?php echo __( '<a href="http://editflow.org/">Edit Flow</a> is produced by <a href="http://danielbachhuber.com/">Daniel Bachhuber</a>, <a href="http://digitalize.ca/">Mo Jangda</a>, and <a href="http://www.scottbressler.com/blog/">Scott Bressler</a>, with special help from <a href="http://andrewspittle.net">Andrew Spittle</a> and <a href="http://andrewwitherspoon.com/">Andrew Witherspoon</a>.', 'edit-flow' ); ?>
+		<br /><?php echo sprintf( __( 'You\'re using Edit Flow version %d.', 'edit-flow' ), EDIT_FLOW_VERSION ); ?>
```

Comment:
Looks like this will need to be a `%s` instead of a `%d`. The `EDIT_FLOW_VERSION` constant can be a string per https://github.com/Automattic/Edit-Flow/blob/master/edit_flow.php#L31